### ISSUE TITLE
Missing Multimodel Pretraining step

### ISSUE THREAD

**Issue by shubhamgarg21** [AILab-CVC/SEED#32]
Hi,

For the paper https://arxiv.org/pdf/2310.01218.pdf , the following is mentioned in pretraining section : 

```
For efficiency, we first train SEED-LLaMA using LoRA [32] tuning and together optimize the
parameters of the embedding layer and decoder head layer due to the added visual codes. We then
merge the parameters of LoRA onto the LLM backbone and fine-tune all parameters except for
the embedding layer.
```

But in the training steps, the part about `fine-tuning all parameters except for the embedding layer` is missing.

**Comment by Cerf-Volant425** [comment]
Same question, could you please add the corresponding script of the fine-tuning part? 
Thanks in advance.
